src/components/dropdown-filter/dropdown-filter.js

```diff
@@ -509,6 +518,20 @@ class DropdownFilter extends Dropdown {
       'data-role': props['data-role']
     };
   }
+
+  onUpArrow = (list, element) => {
+    if (this.props.create) {
+      this.blockBlur = true;
```

Comment:
this is going to block blur even when you click off the list (instead of pressing tab) - doesn't seem like the right solution

Reply:
Good spot. This is essentially imitating the `onMouseEnter` behaviour that is defined in the `Dropdown` component. But you're right, I haven't implemented an equivalent `onMouseLeave` for keyboard users if they don't tab to the create action.

We could define an additional `onBlur` function on the `Dropdown` component itself, that sets `blockBlur` back to `false`.